lib/appsignal/check_in.rb

```diff
@@ -40,6 +51,37 @@ def cron(identifier)
         output
       end
 
+      # Track heartbeat check-ins.
+      #
+      # Track the execution of long-lived processes by sending a heartbeat
+      # check-in.
+      #
+      # @example Send a heartbeat check-in
+      #   Appsignal::CheckIn.heartbeat("main_loop")
+      #
+      # @param identifier [String] identifier of the heartbeat check-in to report.
+      # @param continuous [Boolean] whether the heartbeats should be sent continuously
+      # during the lifetime of the process. Defaults to `false`.
```

Comment:
```suggestion
      #   during the lifetime of the process. Defaults to `false`.
```

Needs to be indented otherwise it becomes part of the description of the entire method:


![2024-09-16 10-11-53 Module AppsignalCheckIn — Documentation for ruby(1)](https://github.com/user-attachments/assets/eb66dd55-c917-4b04-9402-3a3860c0ed31)